strongbox-security/strongbox-authentication-providers/strongbox-default-authentication-provider/src/main/java/org/carlspring/strongbox/authentication/api/impl/xml/CacheManagerAuthenticationCache.java

```diff
@@ -3,11 +3,11 @@
 import org.carlspring.strongbox.data.CacheName;
 import org.springframework.cache.annotation.Cacheable;
 import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
-import org.springframework.security.crypto.password.StandardPasswordEncoder;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 
 public class CacheManagerAuthenticationCache implements AuthenticationCache
 {
-    private StandardPasswordEncoder standardPasswordEncoder = new StandardPasswordEncoder();
```

Comment:
we used `StandardPasswordEncoder` because it's faster than `BCryptPasswordEncoder`. We were aware it's not safe to use `SHA-256`. Maybe we should consider `@SuppressWarnings` here. 

@sbespalov what do you think ?

Reply:
+1: You're right, @fuss86! I remember we had some performance issues with bcrypt!

@sbespalov : I think it was you who implemented the workaround...?

Could you please remind me what the issue was and how we sorted it out?

Also, could you give @adavid9 some hints on how to improve his pull request?

Reply:
we shouldn't drop `StandardPasswordEncoder`, this should be just muted with `@SuppressWarnings`  as Przemyslaw suggested.